crates/holochain_state/src/mutations.rs

```diff
@@ -1021,24 +1021,30 @@ pub fn reschedule_expired(
     Ok(())
 }
 
-pub fn unschedule_fn(
-    txn: &mut Transaction,
-    author: &AgentPubKey,
-    scheduled_fn: &ScheduledFn,
-) -> StateMutationResult<()> {
-    let _ = txn
-        .execute(
-            holochain_sqlite::sql::sql_cell::schedule::DELETE,
-            named_params! {
-                ":zome_name": scheduled_fn.zome_name().to_string(),
-                ":scheduled_fn": scheduled_fn.fn_name().to_string(),
-                ":author" : author,
-            },
-        )
-        .map_err(StateMutationError::from)?;
-    Ok(())
+/// Remove a function from the schedule.
+pub fn unschedule_fn(txn: &mut Transaction, author: &AgentPubKey, scheduled_fn: &ScheduledFn) {
+    match txn.execute(
+        holochain_sqlite::sql::sql_cell::schedule::DELETE,
+        named_params! {
+            ":zome_name": scheduled_fn.zome_name().to_string(),
+            ":scheduled_fn": scheduled_fn.fn_name().to_string(),
+            ":author" : author,
+        },
+    ) {
+        Ok(n) => {
+            tracing::debug!("Unscheduled {n} {scheduled_fn:?} for author {author} in database")
```

Comment:
Nitpick: Does this log message make sense? Won't it print something like:

> Unscheduled 1 some_custom_function for author Bob in database

Maybe something like

```suggestion
            tracing::debug!("Removed {n} scheduled call(s) to '{scheduled_fn:?}' for author '{author}' in database")
```

Reply:
Its currently going to print like:
 > Unscheduled 1 ScheduledFn(ZomeName("my_zome"), FunctionName("my_fn")) for author uhCAk_DCiX-QbpXAOLU8d7xbaFmb3JxVzxXKN8xs_xG9YJtuE0uL8 in database.
 
 I think its good enough. Quotes don't seem necessary and "Unscheduled" is more precise than "Removed" which makes it easier to find and scan in log dump.